setup.py

```diff
@@ -104,7 +104,7 @@ def run(self):
           "docker",
           "pymongo",
           "gdown",
-          "htcondor"
+#          "htcondor"
```

Comment:
Simply commenting this out clearly doesn't work. Will need to investigate some platform dependent way to list dependencies (or a silent fail).